src/Components/EventCreation/Form/Form.js

```diff
@@ -44,6 +53,7 @@ const Form = ({logoutUser}) => {
             className="game-selector"
             id="game"
```

Comment:
There are a lot of ids peppered throughout these components. Ids are considered brittle because they are easily changed and many different pieces of the codebase may depend on them, but they also will hinder your ability to make truly reusable components once you start making large scale applications. Why? Because you may need to reuse a component twice on one page - now you may have to elements with the same id! 

Instead of using ids for your cypress tests, consider using the data attribute. You can read more about the data attribute in the cypress docs [here](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)